src/coreclr/jit/compiler.h

```diff
@@ -10187,6 +10236,56 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     static void PrintAggregateLoopHoistStats(FILE* f);
 #endif // LOOP_HOIST_STATS
 
+#if TRACK_ENREG_STATS
+    class EnregStructStat
```

Comment:
I think this class should be renamed to `EnregisterStats` because it tracks do not enregister for all type of local variables, not just structs.

Reply:
Done